cmd/server-manager/typescript/src/Championship.ts

```diff
@@ -4,11 +4,29 @@ import ClickEvent = JQuery.ClickEvent;
 declare var ChampionshipID: string;
 declare var CanMoveChampionshipEvents: boolean;
 
+interface ACSRDriverRatingGateMet {
+    acsr_driver_rating: ACSRDriverRating;
+    gate_met: boolean;
+}
+
+interface ACSRDriverRating {
+    driver_id: number;
+    skill_rating_grade: string;
+    skill_rating: number;
+    safety_rating: number;
+    num_events: number;
+    is_provisional: boolean;
+}
+
 export namespace Championship {
+
+    declare var championshipID: string;
```

Comment:
this is actually declared above on line 4 (just a different case) - could you use that and rename the const in the HTML?